tests/unit/transformers/test_numerical.py

```diff
@@ -1863,3 +1864,173 @@ def test__reverse_transform_missing_value_replacement_missing_value_replacement_
             call_data,
             rtol=1e-1,
         )
+
+
+class TestLogitScaler:
+    def test___init__super_attrs(self):
+        """Test super() arguments are properly passed and set as attributes."""
+        # Run
+        ls = LogitScaler(
+            missing_value_generation='random',
+            learn_rounding_scheme=False,
+        )
+
+        # Assert
+        assert ls.missing_value_replacement == 'mean'
+        assert ls.missing_value_generation == 'random'
+        assert ls.learn_rounding_scheme is False
+
+    def test___init__(self):
+        """Test super() arguments are properly passed and set as attributes."""
+        # Run
+        ls = LogitScaler(max_value=100.0, min_value=2.0)
+
+        # Assert
+        assert ls.max_value == 100.0
+        assert ls.min_value == 2.0
+
+    def test__validate_logit_inputs(self):
+        """Test validating data against input arguments."""
+        # Setup
+        ls = LogitScaler()
+        data = pd.Series([0.0, 0.1, 0.2, 0.3, 1.0])
+
+        # Run and Assert
+        ls._validate_logit_inputs(data)
+
+    def test__validate_logit_inputs_errors_invalid_value(self):
+        """Test error message contains invalid values."""
+        # Setup
+        ls = LogitScaler()
+        ls.columns = ['column']
+        data = pd.Series([0.0, 0.1, 0.2, 0.3, 1.0, 2.0])
+
+        # Run and Assert
+        expected_msg = re.escape(
+            "Unable to apply logit function to column 'column' due to out of range values ([2.0])."
+        )
+        with pytest.raises(InvalidDataError, match=expected_msg):
+            ls._validate_logit_inputs(data)
+
+    def test__validate_logit_inputs_errors_many_invalid_values(self):
+        """Test error message clips many invalid values."""
+        # Setup
+        ls = LogitScaler()
+        ls.columns = ['column']
+        data = pd.Series([1.0, 1.1, 1.2, 1.3, 2.0, 3.0, 4.0])
+
+        # Run and Assert
+        expected_msg = re.escape(
+            "Unable to apply logit function to column 'column' due to out of range values "
+            '([1.1, 1.2, 1.3, 2.0, 3.0 + 1 more]).'
+        )
+        with pytest.raises(InvalidDataError, match=expected_msg):
+            ls._validate_logit_inputs(data)
+
+    def test__fit(self):
+        """Test the ``_fit`` method validates the inputs."""
+        # Setup
+        ls = LogitScaler()
+        ls._validate_logit_inputs = Mock()
+        data = pd.Series([1.0, 1.1, 1.2, 1.3, 2.0, 3.0, 4.0])
+
+        # Run
+        ls._fit(data)
+
+        # Assert
+        ls._validate_logit_inputs.assert_called_once_with(data)
+
+    @patch('rdt.transformers.numerical.logit')
+    def test__transform(self, mock_logit):
+        """Test the ``transform`` method."""
+        # Setup
+        min_value = (1.0,)
```

Comment:
```suggestion
        min_value = 1.0
```

Reply:
As per this, could we add a validation for the `min_value` and `max_value` inputs, they should be numerical.